central/policy/search/searcher_impl.go

```diff
@@ -24,8 +24,7 @@ var (
 		Field: search.SORTPolicyName.String(),
 	}
 
-	// TODO: ROX-13888 Replace Policy with WorkflowAdministration.
-	policySAC = sac.ForResource(resources.Policy)
+	policySAC = sac.ForResource(resources.WorkflowAdministration)
```

Comment:
perhaps `policySAC` variables should be renamed as well?

Reply:
bit on the fence about this one, since the resource name changes but logicially it's still related to polcieis as the "entity" of this searcher. Given that in other places we didn't change it, let's just leave it for now (as long as it doesn't say "hasWrite...")